docs/retrying-test.adoc

```diff
@@ -57,6 +57,17 @@ The test `aborted` is aborted before (or during) its first execution.
 The first execution is marked as aborted/skipped, containing the underlying cause.
 The test suite as a whole is also marked as aborted/skipped.
 
+[source,java]
```

Comment:
Could you add a few more sentences about `minSuccess`? I think this is a bit too short.

Reply:
I added sections describing the attributes to the documentation. Let me know how you feel about that.